Silicon/Ampere/AmpereSiliconPkg/Library/PlatformBootManagerLib/PlatformBm.h

```diff
@@ -0,0 +1,53 @@
+/** @file
+  Head file for BDS Platform specific code
+
+  Copyright (C) 2015-2016, Red Hat, Inc.
+  Copyright (c) 2004 - 2008, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2016, Linaro Ltd. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PLATFORM_BM_H_
+#define PLATFORM_BM_H_
+
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/DevicePathLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/UefiLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+
+/**
+  Use SystemTable Conout to stop video based Simple Text Out consoles from
+  going to the video device. Put up LogoFile on every video device that is a
+  console.
+
+  @param[in]  LogoFile   File name of logo to display on the center of the
+                         screen.
+
+  @retval EFI_SUCCESS     ConsoleControl has been flipped to graphics and logo
+                          displayed.
+  @retval EFI_UNSUPPORTED Logo not found
+**/
+EFI_STATUS
+EnableQuietBoot (
+  IN  EFI_GUID  *LogoFile
+  );
+
+/**
+  Use SystemTable Conout to turn on video based Simple Text Out consoles. The
+  Simple Text Out screens will now be synced up with all non video output
+  devices
+
+  @retval EFI_SUCCESS     UGA devices are back in text mode and synced up.

```

Comment:
@bexcran 
I'm not sure why it is not removed completely. I still see the UGA is handled in edk2. Should I change UGA to `Graphic Console` or leave it as is since it is just cloned from ArmPkg?

Reply:
Sorry actually you can leave it as is since the same comment is edk2.
```
bcran@delano:/tiano/edk2>  (master)rg "UGA devices are back in text mode"
ArmPkg/Library/PlatformBootManagerLib/PlatformBm.h
46:  @retval EFI_SUCCESS     UGA devices are back in text mode and synced up.

MdeModulePkg/Library/BootLogoLib/BootLogoLib.c
371:  @retval EFI_SUCCESS     UGA devices are back in text mode and synced up.

MdeModulePkg/Include/Library/BootLogoLib.h
29:  @retval EFI_SUCCESS     UGA devices are back in text mode and synced up.

UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.h
101:  @retval EFI_SUCCESS     UGA devices are back in text mode and synced up.
```